src/classes/ManagementSystem.java

```diff
@@ -24,11 +39,30 @@ public Account createAccount(String username, String password, String legalName,
 		return account;
 	}
 	
+	public void createAccountsFromFile(Scanner fileScanner) {
+		String[] accountInformation = null;
+		while(fileScanner.hasNextLine()) {
+			 accountInformation = fileScanner.nextLine().split(";");
+		}
+		
+		System.out.println();
+		if(accountInformation != null && accountInformation.length == 5) {
+			String username = accountInformation[0].replace(";", "");
+			String password = accountInformation[1].replace(";", "");
+			String legalName = accountInformation[2].replace(";", "");
+			String streetAddress = accountInformation[3].replace(";", "");
+			int grossIncome = Integer.parseInt(accountInformation[4]);
+			Account account = this.createAccount(username, password, legalName, streetAddress, grossIncome);
+			this.accounts.add(account);

```

Comment:
Doesn't `createAccount()` already add the account to `accounts` and increment `numAccounts`?